pundix/chain.json

```diff
@@ -0,0 +1,173 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "Pundi X Chain",
+  "chain_id": "PUNDIX",
+  "website": "https://pundix.com",
+  "pretty_name": "Pundi X Chain",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "px",
+  "daemon_name": "pundixd",
+  "node_home": "$HOME/.pundix",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "ibc/55367B7B6572631B78A93C66EF9FDFCE87CDE372CC4ED7848DA78C1EB1DCDD78",
+        "fixed_min_gas_price": 2000000000000,
+        "low_gas_price": 2000000000000,
+        "average_gas_price": 2500000000000,
+        "high_gas_price": 3000000000000
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "ibc/55367B7B6572631B78A93C66EF9FDFCE87CDE372CC4ED7848DA78C1EB1DCDD78"
```

Comment:
And IBC token as a staking token is highly unusual. Just want to double check that the staking token isn't native to this chain?

Reply:
Yes, this is indeed strange, but since this token was originally created on Ethereum and came across the chain via fxcore ibc, in order to follow the ibc protocol, in the end, we used the ibc token as a staking token.